data/sql/updates/pending_db_world/rev_1598878078501049600.sql

```diff
@@ -0,0 +1,39 @@
+INSERT INTO `version_db_world` (`sql_rev`) VALUES ('1598878078501049600');
+/*
+ * Dungeon: Dire Maul (East)
+ * Update by Knindza | <www.azerothcore.org>
+ * Copyright (C) <www.shadowburn.net> & <www.lichbane.com>
+*/
+
+/* REGULAR */ 
+UPDATE `creature_template` SET `mindmg` = 437, `maxdmg` = 591, `DamageModifier` = 1.03 WHERE `entry` = 11464;
+UPDATE `creature_template` SET `mindmg` = 461, `maxdmg` = 624, `DamageModifier` = 1.03 WHERE `entry` = 11462;
+UPDATE `creature_template` SET `mindmg` = 452, `maxdmg` = 617, `DamageModifier` = 1.03 WHERE `entry` = 13196;
+UPDATE `creature_template` SET `mindmg` = 516, `maxdmg` = 696, `DamageModifier` = 1.03 WHERE `entry` = 13021;
+UPDATE `creature_template` SET `mindmg` = 312, `maxdmg` = 461, `DamageModifier` = 1.03 WHERE `entry` = 13022;
+UPDATE `creature_template` SET `mindmg` = 333, `maxdmg` = 471, `DamageModifier` = 1.03 WHERE `entry` = 11455;
+UPDATE `creature_template` SET `mindmg` = 470, `maxdmg` = 637, `DamageModifier` = 1.03 WHERE `entry` = 11451;
+UPDATE `creature_template` SET `mindmg` = 461, `maxdmg` = 628, `DamageModifier` = 1.03 WHERE `entry` = 11454;
+UPDATE `creature_template` SET `mindmg` = 446, `maxdmg` = 595, `DamageModifier` = 1.03 WHERE `entry` = 13197;
+UPDATE `creature_template` SET `mindmg` = 478, `maxdmg` = 624, `DamageModifier` = 1.03 WHERE `entry` = 14349;
+UPDATE `creature_template` SET `mindmg` = 452, `maxdmg` = 601, `DamageModifier` = 1.03 WHERE `entry` = 11456;
+UPDATE `creature_template` SET `mindmg` = 446, `maxdmg` = 604, `DamageModifier` = 1.03 WHERE `entry` = 11453;
+UPDATE `creature_template` SET `mindmg` = 449, `maxdmg` = 607, `DamageModifier` = 1.03 WHERE `entry` = 11452;
+UPDATE `creature_template` SET `mindmg` = 347, `maxdmg` = 437, `DamageModifier` = 1.03 WHERE `entry` = 13276;
+UPDATE `creature_template` SET `mindmg` = 446, `maxdmg` = 604, `DamageModifier` = 1.03 WHERE `entry` = 11457;
+UPDATE `creature_template` SET `mindmg` = 489, `maxdmg` = 660, `DamageModifier` = 1.03 WHERE `entry` = 13285;
+UPDATE `creature_template` SET `mindmg` = 456, `maxdmg` = 615, `DamageModifier` = 1.03 WHERE `entry` = 11461;
+UPDATE `creature_template` SET `mindmg` = 452, `maxdmg` = 609, `DamageModifier` = 1.03 WHERE `entry` = 11465;
+
+/* BOSS */
+UPDATE `creature_template` SET `type_flags` = 4, `mindmg` = 601, `maxdmg` = 702, `DamageModifier` = 1.01 WHERE `entry` = 14354;
+UPDATE `creature_template` SET `type_flags` = 4, `mindmg` = 602, `maxdmg` = 704, `DamageModifier` = 1.01 WHERE `entry` = 14327;
+UPDATE `creature_template` SET `type_flags` = 4, `mindmg` = 608, `maxdmg` = 714, `DamageModifier` = 1.01 WHERE `entry` = 13280;
+UPDATE `creature_template` SET `type_flags` = 4, `mindmg` = 619, `maxdmg` = 728, `DamageModifier` = 1.01 WHERE `entry` = 11490;
+UPDATE `creature_template` SET `type_flags` = 4, `mindmg` = 615, `maxdmg` = 712, `DamageModifier` = 1.01 WHERE `entry` = 11492;
```

Comment:
you should edit the flags in a different way https://www.azerothcore.org/wiki/Dealing-with-SQL-files#sql-basics

Reply:
Sorted xP